lib/elixir/lib/macro.ex

```diff
@@ -692,6 +692,11 @@ defmodule Macro do
     fun.(ast, "&(" <> to_string(arg, fun) <> ")")
   end
 
+  # left not in right
+  def to_string({:not, _, [{:in, meta, args}]}, fun)  do
+    to_string({:"not in", meta, args}, fun)
```

Comment:
If we do this, we will eventually invoke `fun.({:"not in", meta, args})` and therefore :"not in" will leak to the user. Instead, we should call `fun.({:not, not_meta, [{:in, meta, args}]})` right here, then translate the args and build the "left not in right" here directly.